src/components/DeleteComment.jsx

```diff
@@ -0,0 +1,18 @@
+import { deleteComment } from "../utils/api";
+
+const DeleteComment = ({ comment_id, setDeleted, setComments }) => {
+  const handleClick = () => {
+    deleteComment(comment_id).then(() => {
+      setDeleted(true);
+      setComments(comments => [...comments]);
+    });
+  };
+
+  return (
+    <button onClick={handleClick} className="button--delete__comment">
+      X
+    </button>
```

Comment:
This button is visually nice, but not very accessible. All you need to do is give it an attribute like an `aria-label="delete comment"` so that a screen reader can understand what it is better!

Reply:
Okay, I'll get onto it! Thanks for the feedback!